Makefile

```diff
@@ -15,10 +15,11 @@ docker-publish:
 
 .PHONY: run-docker
 run-docker:
-	docker run -v $(PWD)/output/:/usr/src/app/output/ ga4gh/drs-compliance-suite:${DOCKER_TAG} \
+	docker run -d -v $(PWD)/output/:/usr/src/app/output/ -p 57568:57568 \
+	ga4gh/drs-compliance-suite:${DOCKER_TAG} \
 	--server_base_url "http://host.docker.internal:8089/ga4gh/drs/v1" \
 	--platform_name "ga4gh starter kit drs" --platform_description "GA4GH reference implementation of DRS specification" \
-	--auth_type "none" --report_path "./output/test-report.json"
```

Comment:
Since auth_type is now being passed from the config file does that mean it is no longer required for command line argument?

Reply:
thats correct